dashboard/test/controllers/omniauth_callbacks_controller_test.rb

```diff
@@ -54,7 +54,7 @@ class OmniauthCallbacksControllerTest < ActionController::TestCase
         'info',
         nickname: '',
         name: {'first' => 'Hat', 'last' => 'Cat'},
-        email: 'first_last@clever_teacher.xx',
+        email: 'first_last@clever-teacher.xx',
```

Comment:
Underscores are not allowed in the domain part of the email, in the email spec.

I checked and we have 7 user email addresses with underscores in that place, most never logged in or only logged in once. In all cases, we were unable to send them emails (`sent_at` is 0, which means it gave up due to an unrecoverable send error, such as a badly formatted address in `poste_deliveries`)